src/libs/actions/Report.js

```diff
@@ -198,6 +204,33 @@ function setLocalLastRead(reportID, sequenceNumber) {
     });
 }
 
+/**
+ * Remove a specific action from a report by index.
+ *
+ * @param {Number} reportID
+ * @param {String} actionID
+ */
+function removeLocalAction(reportID, actionID) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, {
+        [actionID]: null,
+    });
+}
+
+/**
+ * Remove all temporary actions from a local report.
+ *
+ * @param {Number} reportID
+ */
+function removeTemporaryActions(reportID) {
```

Comment:
What's the difference between a "temporary" action and a "local" action (from the method above)?

Reply:
A "local action" would be _any_ action in storage. The temporary actions are specifically actions that need to be paired with a server action or removed.

Reply:
OK, I think the term "local" can be a little confusing because I was thinking it was referring to something like a local variable. The temporary actions make sense, but maybe it would be better to refer to them specifically as `optimisticActions`?

For "local actions", maybe those can be referred to more specifically as "actions in storage" (looks like you suggested this above) or "actions in Onyx" (sux if we ever rename Onyx)?

Reply:
That makes sense to me. Let me update any comments left over that are still referencing "local".